package.json

```diff
@@ -71,6 +71,8 @@
     "eslint-plugin-promise": "^5.1.1",
     "eslint-plugin-react": "^7.27.0",
     "eslint-plugin-react-hooks": "^4.3.0",
+    "fs-extra": "^10.1.0",
```

Comment:
Hm, why do we need this dependency?

Reply:
@vojtechszocs Sorry, maybe that was pushed up by accident. It's not needed and I've removed it. Thanks!